ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -560,6 +560,12 @@ private class DisplayURLLabel: UILabel {
         updateText()
         updateTextSize()
         detectLanguageForNaturalDirectionClipping()
+
+        if let text = text {
+          isWebScheme = ["http", "https"].contains(URL(string: text)?.scheme ?? "")
+        } else {
+          isWebScheme = false
+        }
```

Comment:
You set this after calling `detectLanguageForNaturalDirectionClipping` which relies on it when updating the `gradientLayer`

Reply:
You probably need to set this before `updateText`

Reply:
Oh man. I see. I should have just kept it how it was before.

I'll update it and retest all the flaws to make sure everything still works.